Source/JavaScriptCore/assembler/CPU.h

```diff
@@ -124,6 +124,15 @@ constexpr bool isRISCV64()
 #endif
 }
 
+constexpr bool isJSValue3264()
```

Comment:
What info is this mean to convey?  Why not use `!is64Bit()` instead?

Reply:
My first thought is that this should be named `is32Bit()` instead.  But then, I realized you can just do `!is64Bit()`.  Hence, I asked why this is needed.